### DISCUSSION TITLE
How to disable type checking for a python package?

### DISCUSSION THREAD

**Discussion by ahmedbilal** [microsoft/pyright#4335] [General]
I want to disable type checking for a python package. For instance, my project is a django based and there is a dependency i.e django-environ that doesn't play well with typings. Can I disable type checking for django-environ? or any other package?

Thanks

**Comment by erictraut** [comment]
You can create a local stub file (which are typically located in a project-local directory called "typings") that tells pyright to assume that any symbols imported from `django-environ` have the type `Any`.

Here's what that would look like:
```python
# typings/django-environ/__init__.pyi
def __getattr__(name: str) -> Any: ...
```

With this technique, you have the option of adding additional type information for specific classes or functions in `django-environ`. For example, if you want completion suggestions for certain functions, you can add a typed signature to that function in the stub file.

**Reply by ahmedbilal** [reply]
@erictraut Thanks. It worked!
